### ISSUE TITLE
Not require enum for tryLaunch

### ISSUE THREAD

**Issue by rafaelnobrepd** [evermeer/UITestHelper#1]
Hi! Great job on this helper lib, looks promising to my intermitent failures on Travis.
I have an issue tho with the tryLaunch requiring an enum to work. Firstly, if I don't need any custom launch args at all I still have to provide a populated non-optional array. Second, a UITest target cannot reference the application bundle, so this enum has to be replicated on each UITesting target AND the app (if I want to act on those values accordingly). It would suffice to make it an optional [String].

**Comment by evermeer** [comment]
True, in version 0.2.0 you have to add the enum to both targets (you don't have to replicate the code, you only have to add it to both targets) And you are right that it would be nice to be able to start without the need of any enum.

I just published version 0.3.0 that solves this problem for you. You can now launch the app with code like this:

```
        self.tryLaunch(["NoneEnumOption"]) // For if you don't want an Enum that is both in you test target and application
        self.tryLaunch() // For if you don't need any options at all
```

**Comment by rafaelnobrepd** [comment]
Thanks for the update! And you're absolutely right on adding the enum file to needed targets 👍
